messageapp/routes/index.routes.js

```diff
@@ -0,0 +1,32 @@
+const router = require("express").Router();
+const ApiHandler = require('./../services/APIHandler')
+
+const messageHandler = new ApiHandler()
+
+router.post("/messages", (req, res, next) => {
+
+  const { destination, message } = req.body
+
+  if (!destination && !message) {
+    res.status(400).json({ message: "You should not pass empty object" })
```

Comment:
I liked you that you use 400 for status code, do you think that 422 would make more sense?